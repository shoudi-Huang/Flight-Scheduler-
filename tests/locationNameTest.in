location
loCaTiOn NewDelhi
location Mumbai 10 ops
location add NewDelhi 28.527 77.0688988 -0.123
location add Mumbai 19.08 72.741 0.4
LOCATION NewDelhi
LOcATion Mumbai isk
location add AliceSprings -23.7206896 133.7977352 0.056
location add NewYork 40.6971494 -74.2598661 -0.09853
location aliCEsPRings
location Beijing 1
location NEWyOrk qpp lk shi 10
exit
