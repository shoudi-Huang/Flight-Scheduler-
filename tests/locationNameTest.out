User: Usage:
LOCATION <name>
LOCATION ADD <name> <latitude> <longitude> <demand_coefficient>
LOCATION IMPORT/EXPORT <filename>

User: Invalid location name.

User: Invalid location name.

User: Successfully added location NewDelhi.

User: Successfully added location Mumbai.

User: Location:    NewDelhi
Latitude:    28.527000
Longitude:   77.068899
Demand:      -0.1230

User: Location:    Mumbai
Latitude:    19.080000
Longitude:   72.741000
Demand:      +0.4000

User: Successfully added location AliceSprings.

User: Successfully added location NewYork.

User: Location:    AliceSprings
Latitude:    -23.720690
Longitude:   133.797735
Demand:      +0.0560

User: Invalid location name.

User: Location:    NewYork
Latitude:    40.697149
Longitude:   -74.259866
Demand:      -0.0985

User: Application closed.
